state-chain/pallets/cf-pools/src/lib.rs

```diff
@@ -1520,6 +1528,7 @@ impl<T: Config> Pallet<T> {
 				.try_map(|(asset, collected_fees)| {
 					AssetAmount::try_from(collected_fees).map_err(Into::into).and_then(
 						|collected_fees| {
+							T::LpBalance::record_fees(lp, collected_fees, asset);
```

Comment:
@AlastairHolmes that's the place where I have to add record the fees for range orders, right?

Reply:
Sorry not sure how I could have missed that